### ISSUE TITLE
Can't repeat the metric on peta

### ISSUE THREAD

**Issue by lumiaomiao** [chufengt/iccv19_attribute#25]
Hi, I can't repeat the paper metric on peta with the code, and I have send train log to your email. Could you give me some help or advice? I am trying on pa-100k now.

**Comment by valencebond** [comment]
@lumiaomiao , i only get `ma: 0.7875,  Acc: 0.7810, Prec: 0.8733, Rec: 0.8616, F1: 0.8630` in PA100k, can your achieve the paper performance ?

**Comment by chufengt** [comment]
@lumiaomiao @valencebond 

Hi, there are some tips for re-implementation:

1. I have uploaded the data lists, pre-trained models into Baidu Cloud.
(link: https://pan.baidu.com/s/1sE5aD2JsO7Zmkge20M59Hw  password: pmqi)

2. A warm-up trick mentioned in #7, you can train a simple BN-Inception baseline model firstly, and initialize the inception_iccv model with the baseline parameters.

**Comment by zhangjingling** [comment]
> @lumiaomiao , i only get `ma: 0.7875, Acc: 0.7810, Prec: 0.8733, Rec: 0.8616, F1: 0.8630` in PA100k, can your achieve the paper performance ?



您好，请问能否把data list  和  预训练模型重新上传一下，原作者的我下载不了，过期了。多谢哈

**Comment by chufengt** [comment]
> > @lumiaomiao , i only get `ma: 0.7875, Acc: 0.7810, Prec: 0.8733, Rec: 0.8616, F1: 0.8630` in PA100k, can your achieve the paper performance ?
> 
> 您好，请问能否把data list 和 预训练模型重新上传一下，原作者的我下载不了，过期了。多谢哈

链接: https://pan.baidu.com/s/1UoTABgR4NZVb2OUPrZVuDg  密码: 3ub4
近期会整理一下直接放上来
